Gruntfile.js

```diff
@@ -108,99 +125,100 @@ module.exports = function (grunt) {
-    compress = compress === undefined ? true : compress;
+    const THEME_DIR = path.join(DIST_DIR, '/', theme);
+    const isValidTheme = grunt.file.exists(path.join(THEME_DIR, '/_variables.scss')) && grunt.file.exists(path.join(THEME_DIR, '/_bootswatch.scss'));
 
-    var isValidTheme = grunt.file.exists('dist/' + theme, '_variables.scss') && grunt.file.exists('dist/' + theme, '_bootswatch.scss');
-
-     // cancel the build (without failing) if this directory is not a valid theme
+    // cancel the build (without failing) if this directory is not a valid theme
     if (!isValidTheme) {
       return;
     }
-    var concatSrc = 'build/scss/build.scss';
-    var concatDest = 'dist/' + theme + '/build.scss';
-    var scssSrc = 'dist/' + theme + '/build.scss';
-    var scssDest = grunt.config.get('builddir') + '/' + theme + '/bootstrap.css';
-
-    var dist = {src: concatSrc, dest: concatDest};
-    var distPost = {src: scssDest, dest: scssDest};
-    grunt.config.set('concat.dist', dist);
-    grunt.config.set('postcss.dist', distPost);
-    var files = {};
-    files[scssDest] = scssSrc;
-    grunt.config.set('sass.dist.files', files);
-    grunt.config.set('sass.dist.options.implementation', sass);
-    grunt.config.set('sass.dist.options.outputStyle', 'expanded');
-    grunt.config.set('sass.dist.options.precision', 6);
+
+    const concatSrc = path.join(BUILD_DIR, '/scss/build.scss');
+    const concatDest = path.join(THEME_DIR, '/build.scss');
+    const scssSrc = path.join(THEME_DIR, '/build.scss');
+    const scssDest = path.join(THEME_DIR, '/bootstrap.css');
+    const scssDestMin = path.join(THEME_DIR, '/bootstrap.min.css');
+    const concatDist = {
+      src: concatSrc,
+      dest: concatDest
+    };
+    const sassDist = {
+      src: scssSrc,
+      dest: scssDest
+    };
+    const postCssDist = {
+      src: scssDest,
+      dest: scssDest
+    };
+    const cleanCssDist = {
+      src: scssDest,
+      dest: scssDestMin
+    };
+
+    grunt.config.set('concat.dist', concatDist);
+    grunt.config.set('sass.dist', sassDist);
+    grunt.config.set('postcss.dist', postCssDist);
+    grunt.config.set('copy.css.files.0.cwd', THEME_DIR);
+    grunt.config.set('copy.css.files.0.dest', path.join(DOCS_DEST, theme));
+    grunt.config.set('cssmin.dist', cleanCssDist);
 
     grunt.task.run([
       'concat',
       'sass:dist',
       'postcss:dist',
       'clean:build',
-      compress ? 'compress:' + scssDest + ':' + grunt.config.get('builddir') + '/' + theme + '/bootstrap.min.css' : 'none',
+      'cssmin:dist',
       'copy:css'
     ]);
   });
 
-  grunt.registerTask('compress', 'minify a built css file', function(fileSrc, fileDst) {
-    var distPost = {src: fileSrc, dest: fileDst};
-    grunt.config.set('cssmin.dist', distPost);
-    grunt.log.writeln('Minifying file ' + fileSrc);
-    grunt.task.run(['cssmin:dist']);
-  });
-
   grunt.registerMultiTask('swatch', 'build a theme', function() {
```

Comment:
I don't quite get this task. Is it even used?

Reply:
It was used once upon the time. Might have been v2!

Reply:
We should remove it then, since it's confusing :)